src/main/java/com/google/gfx/model/Match.java

```diff
@@ -0,0 +1,51 @@
+package com.google.gfx.model;
+
+/**
+ * A match to an existing order, generated by a user responding to a buy/sell order or by an cron
+ * task that matches existing orders periodically.
+ */
+public class Match {
+
+  /**
+   * The buy order being matched.
+   */
+  private Order buyOrder;
+
+  /**
+   * The sell order being matched.
+   */
+  private Order sellOrder;
+
+  /**
+   * The exchange rate that applied to the currency being bought results in the amount of the
+   * currency being sold.
+   */
+  private float fxRateBuyToSell;
+
+  /**
+   * The buyer's response. {@code true} if accepted, {@code false} is declined, or {@code null} if
+   * non responded yet.
+   */
+  private Boolean buyerResponse;
+
+  /**
+   * The seller's response. {@code true} if accepted, {@code false} is declined, or {@code null} if
+   * non responded yet.
+   */
+  private Boolean sellerResponse;
+
+  /**
+   * The final transactioned amount as part of this trade.
+   */
+  private float amount;
```

Comment:
We [shouldn't use float to represent money](http://stackoverflow.com/questions/3730019/why-not-use-double-or-float-to-represent-currency) due to rounding errors. The same goes for the fx rate.